modules/shared/system_module_version.mk

```diff
@@ -34,5 +34,12 @@ endif
 # Bump by 1 for every updated bootloader image for a release with the same v0.x.* base.
 BOOTLOADER_VERSION ?= 301
 
-# the version of the bootloader that the system firmware requires
+# The version of the bootloader that the system firmware requires
+# NOTE: this will force the device into safe mode until this dependency is met, which is why
+# this version usually lags behind the current bootloader version, to avoid non-mandatory updates.
+ifneq (,$(filter $(PLATFORM_GEN),2))
```

Comment:
`ifneq ($(PLATFORM_GEN),2)` ? It's not a list anymore. :smile:

Reply:
Totally